app/controllers/users_controller.rb

```diff
@@ -0,0 +1,7 @@
+class UsersController < ApplicationController
+  def create
+    @user = User.new
+  end
+
+
+end
```

Comment:
I'm surprised you created a UsersController when the user stories (and your test) only ask you to visit a book_path(book).